sources/LocalizationEditor/Resources/zh-Hant.lproj/Localizable.strings

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Please make the file looks the same as for other languages, so header and get rid of the excessive newlines.